libs/community/langchain_community/document_loaders/url_playwright.py

```diff
@@ -59,7 +59,7 @@ async def evaluate_async(
 class UnstructuredHtmlEvaluator(PlaywrightEvaluator):
     """Evaluate the page HTML content using the `unstructured` library."""
 
-    def __init__(self, remove_selectors: Optional[List[str]] = None):
+    def __init__(self, remove_selectors: Optional[List[str]] = None, keep_only_selectors: Optional[List[str]] = None):
```

Comment:
This name is misleading - I was expecting it to be a boolean.

How about `scraped_selectors` or something similar?